.github/workflows/test-trezor-user-env-link-e2e.yml

```
@@ -0,0 +1,38 @@
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - ".github/workflows/test-trezor-user-env-link-e2e.yml"
+      - "packages/trezor-user-env/**"
+  workflow_dispatch:
+
+concurrency:
+  cancel-in-progress: true
+
+e2e-test-trezor-user-env-link:
```

actionlint (1.7.4)

unexpected key "e2e-test-trezor-user-env-link" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)